### PR TITLE
Separate english locales

### PR DESCRIPTION
**Pull request by razzeee**
This might cause weblate to have problems, as we're removing the source language.

### CODE REVIEW

File: frontend/next-i18next.config.js [hunk 1]
```diff
@@ -2,9 +2,10 @@
 
 module.exports = {
   i18n: {
-    defaultLocale: "en",
+    defaultLocale: "en_GB",
```
Comment (wjt) [inline frontend/next-i18next.config.js:5]: GB as default! :tada:

File: frontend/pages/statistics.tsx [hunk 2]
```diff
@@ -64,7 +65,7 @@ const Statistics = ({ stats }: { stats: Statistics }): JSX.Element => {
                 content: {
                   type: "text",
                   text: stats.downloads?.toLocaleString(
-                    i18n.language.substring(0, 2),
+                    getIntlLocale(i18n.language),
```
Comment (wjt) [inline frontend/pages/statistics.tsx:68]: but I don't think this is going to work. The web platfrom uses the hyphenated form, `en-GB`. But this function appears to return the underscored form.

**Review by wjt — COMMENTED** [review]
```
new Date().toLocaleDateString("en_GB", {dateStyle: 'long'})
Uncaught RangeError: Incorrect locale information provided
    at Date.toLocaleDateString (<anonymous>)
    at <anonymous>:1:12
(anonymous) @ VM95:1
```

File: frontend/next-i18next.config.js [hunk 3]
```diff
@@ -2,9 +2,10 @@
...
     locales: [
-      "en",
+      "en_GB",
+      "en_US",
```
Comment (wjt) [inline frontend/next-i18next.config.js:8]: In my experience it is very unusual to have a specific US locale. More typically there is an 'en' one and then some variations like GB, CA, AU, ...

File: frontend/pages/statistics.tsx (hunk 2 above)
Comment (wjt) [reply frontend/pages/statistics.tsx:68]: Oh I see, this is actually returning a `Locale` object.

File: frontend/next-i18next.config.js (hunk 1 above)
Comment (razzeee) [reply frontend/next-i18next.config.js:5]: Yeah, looking at the formats, I think most of the world will have a much better time with those number/date formats.

File: frontend/next-i18next.config.js (hunk 3 above)
Comment (razzeee) [reply frontend/next-i18next.config.js:8]: I'm not sure why that would matter, I think the website will route you to the US version if your browser says you would like that.

But yeah, reordering would probably help us avoid the weblate breakage.

File: frontend/next-i18next.config.js (hunk 1 above)
Comment (razzeee) [reply frontend/next-i18next.config.js:5]: Probably won't last, if we go for the less invasive version for now.

**Pull request merged**